plonky2x/core/src/backend/prover/remote.rs

```diff
@@ -61,18 +61,26 @@ impl RemoteProver {
             .submit::<L, D>(request)
             .expect("failed to submit proof request");
 
-        // Wait for the proof to be generated.
-        const MAX_RETRIES: usize = 500;
+        // Default timeout for a proof is 60 minutes. Users can override this value by
+        // setting the PROOF_TIMEOUT_MINS environment variable.
+        const DEFAULT_PROOF_TIMEOUT_MINS: u64 = 60;
+        let proof_timeout_mins =
+            env::var("PROOF_TIMEOUT_MINS").unwrap_or(DEFAULT_PROOF_TIMEOUT_MINS.to_string());
```

Comment:
nit: I think minutes is pretty strange as a configuration variable, bullish defining it in seconds